### ISSUE TITLE
H1 Header Usage in Subsections

### ISSUE THREAD

**Issue by bengm** [davetron5000/brutalist-web-design#11]
The page uses H1 elements within each section. Each of the following are h1:

> Guidelines for Brutalist Web Design [page/site title]
> ...
> Detailed Guidelines for Brutalist Web Design [major section heading]
> Only hyperlinks and buttons respond to clicks. [subheading in nested section]
> Hyperlinks are underlined and buttons look like buttons. [subheading in nested section]

I couldn't find clear guidance after a bit of research. The examples in the following would align to starting off those subsections w/ a lower level heading (e.g. h2):
https://www.w3.org/TR/html52/sections.html#headings-and-sections

My understanding is that screen readers and potentially other agents will properly deal w/ h1-n levels, but I'm not sure how robust support is for section nesting. (I'd love to understand that better).

Bottom line: I *think* that nesting the header levels (not all h1) would make this more usable, accessible, and perhaps semantically correct. Disclaimer: I'm not claiming expertise to say this authoritatively. Honestly, this issue is more intended as discussion than obvious flaw, as I don't know the best answer (and no offense taken if this issue is closed). If this is a valid change, I'm happy to take the assignment to draft a PR.

**Comment by davetron5000** [comment]
Yeah, I went back and forth on this a lot.  I eventually decided to `<h1>` inside each section because the default browser styling renders it differently, i.e. an `<h1>` inside a `<section>` is a smaller font than an `<h1>` outside it.  This said to me that the browser treats them differently and can understand that `<h1>` in context.

But that's an agreeably flimsy argument :)

Given that the header has an `<h1>`, `<h2>`, and `<h3`>, do you think each section should be an `<h4>`?  I don't think that would be wrong per se.  Might even be better?  I just don't have a good sense of what a screen reader would do and how a person using one would comprehend it.

**Comment by davetron5000** [comment]
Closing as old.  Feel free to re-open if you have additional thoughts or information.

**Comment by davetron5000** [comment]
I changed my mind and have changed this to have only one `<h1>`  and the rest of the `<h*>` tags generally form an outline.  See b01ac5e5519b612da34518861bdd35c3248592b6
